### PR TITLE
Set up Blazer 

### PR DESCRIPTION
**Pull request by TobyRet**
### Ticket 
#821 

### Changes
- Add Blazer and configuration

### Notes
- Will not be available on live / prod
- Can be accessed at '/admin/blazer' - you need to sign in as a DFE / admin person to access

blazer_on_review_app

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,3 +8,4 @@ ENABLE_PERSONAS=true
 ENABLE_MIGRATION_TESTING=true
 ENABLE_SCHOOLS_INTERFACE=true
 TEST_GUIDANCE=true
+ENABLE_BLAZER=true
```
Comment (TobyRet) [inline .env.example:11]: This is to allow us to restrict which environment Blazer is accessed (I'm assuming we do not want on live / production right now)

File: config/blazer.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# see https://github.com/ankane/blazer for more info
+
+data_sources:
+  main:
+    url: <%= ENV["BLAZER_DATABASE_URL"] %>
```
Comment (TobyRet) [inline config/blazer.yml:5]: Do we have a url for the read-only db?

File: app/controllers/application_controller.rb [hunk 3]
```diff
@@ -8,6 +8,12 @@ class ApplicationController < ActionController::Base
 
 private
 
+  def require_persona_admin
```
Comment (TobyRet) [inline app/controllers/application_controller.rb:11]: This seems a bit pointless whilst we are using personas? Should we set separate Basic Auth credentials instead?
[27bc33a](https://github.com/ankane/blazer?tab=readme-ov-file#basic-authentication)

File: app/controllers/application_controller.rb [hunk 4]
```diff
@@ -8,6 +8,11 @@ class ApplicationController < ActionController::Base
 
 private
 
+  def require_admin
+    # This method is used by Blazer to restrict access. See config/blazer.yml
+    redirect_to('/sign-in') unless Admin::Access.new(current_user).can_access?
```
Comment (TobyRet) [inline app/controllers/application_controller.rb:13]: @tonyheadford - can you confirm that this check for admin users will suffice for the foreseeable?

Comment (tonyheadford) [reply app/controllers/application_controller.rb:13]: Yes I think so

**Review by tonyheadford — APPROVED** [review]

**Pull request merged**